mne/io/egi/tests/test_egi.py

```diff
@@ -288,4 +292,37 @@ def test_io_egi_crop_no_preload():
     assert_allclose(raw._data, raw_preload._data)
 
 
+@pytest.mark.parametrize('idx, cond, signals, bads', [
+    (0, 'Category 1', egi_txt_evoked_cat1_fname,
+     ['E8', 'E11', 'E17', 'E28', 'ECG']),
+    (1, 'Category 2', egi_txt_evoked_cat2_fname,
+     ['E257', 'EMG'])
+])
+def test_io_egi_evokeds_mff(idx, cond, signals, bads):
+    """Test reading evoked MFF file."""
+    # Test reading all conditions from evokeds
+    evokeds = read_evokeds_mff(egi_mff_evoked_fname)
+    assert_equal(len(evokeds), 2)
+    # Test reading evoked data from single condition
+    evoked_cond = read_evokeds_mff(egi_mff_evoked_fname, condition=cond)
+    evoked_idx = read_evokeds_mff(egi_mff_evoked_fname, condition=idx)
+    assert_equal(evoked_cond.comment, cond)
+    assert_equal(evoked_idx.comment, cond)
+    # Check signal data
+    data = np.loadtxt(signals, ndmin=2).transpose() * 1e-6  # convert to volts
+    assert_allclose(evoked_cond.data, data, atol=1e-6)
+    assert_allclose(evoked_idx.data, data, atol=1e-6)
+    # Check info
+    assert_equal(evoked_cond.info, evoked_idx.info)
+    assert_equal(evoked_cond.info['description'], cond)
+    assert_equal(evoked_cond.info['bads'], bads)
```

Comment:
for builtin types we now prefer to let pytest do nice inspection for us, and this is a list, so
```suggestion
    assert evoked_cond.info['bads'] == bads
```